scripts/process_new_format.py

```
@@ -47,71 +52,85 @@
-    '"As If": This was added to the FAQ (v.1.7, March 2020) and then amended (v.1.8, October 2020). You can read the October ruling on the ArkhamDB rules page here. (I\'m adding a hyperlink rather than retyping the rules in case in future the ruling is changed or amended - at that point, the rules page will be updated and all ArkhamDB FAQ entries will link to the correct ruling.)',
-]
+# RULING_REMOVAL_PATTERNS and RULING_STRIP_PATTERNS moved to constants.py
 
 
 def load_faqs(faqs_path: Path) -> dict[str, dict[str, str]]:
+    # Path validation and loading logic
+    """
+    Load FAQ data from a JSON file after validating the file's existence, type, extension, and non-emptiness.
+    
+    Parameters:
+    	faqs_path (Path): Path to the FAQ JSON file.
```

Ruff (0.11.9)

Indentation contains mixed spaces and tabs

(E101)